packages/wallets/src/components/WalletTourGuide/WalletTourGuideSettings.tsx

```diff
@@ -13,21 +19,141 @@ export const tourStepConfig: Step[] = [
         ),
         disableBeacon: true,
         disableOverlayClose: true,
+        placement: 'bottom',
+        spotlightPadding: 0,
+        styles: { spotlight: { borderRadius: '1.6rem 1.6rem 0rem 0rem' } },
         target: '.wallets-accordion__header:has(+ .wallets-accordion__content--visible)',
         title: (
             <WalletText color='red' size='sm' weight='bold'>
                 Wallets
             </WalletText>
         ),
     },
+    // Actions
     {
-        content: <WalletText size='sm'>Step 2</WalletText>,
+        content: isDemoWallet ? (
+            <WalletText size='sm'>
+                Perform Transfer and Reset balance using your Demo Wallet. You can also view your Demo wallet&apos;s
+                transaction history.
+            </WalletText>
+        ) : (
+            <WalletText size='sm'>
+                Perform deposits, withdrawals, and fund transfers using your Wallet. You can also view your
+                Wallet&apos;s transaction history.
+            </WalletText>
+        ),
         disableBeacon: true,
         disableOverlayClose: true,
-        target: '.wallets-accordion__header:has(+ .wallets-accordion__content--visible)',
+        placement: 'bottom',
+        spotlightPadding: 4,
+        styles: {
+            spotlight: {
+                borderRadius: '6.4rem',
+            },
+        },
+        target: '.wallets-header__actions',
         title: (
             <WalletText color='red' size='sm' weight='bold'>
-                Wallets
+                Wallet actions
+            </WalletText>
+        ),
+    },
+    // CFDs
+    {
+        content: hasMT5Account ? (
+            <WalletText size='sm'>
+                This is your CFDs trading account. Click Transfer to move funds between your Wallet and trading account.
+            </WalletText>
+        ) : (
+            <WalletText size='sm'>
+                This is your CFDs trading account. Click Get to create the trading account you desire for trading.
+            </WalletText>
+        ),
+        disableBeacon: true,
+        disableOverlayClose: true,
+        placement: 'right',
+        spotlightPadding: 5,
+        // styles: { spotlight: { borderRadius: '0.6rem' } },
+        styles: { spotlight: { borderRadius: '0.4rem', paddingLeft: '0.5rem', paddingRight: '0.5rem' } },
+        target: '.wallets-mt5-list__content .wallets-trading-account-card',
+        title: (
+            <WalletText color='red' size='sm' weight='bold'>
+                CFDs trading accounts
+            </WalletText>
+        ),
+    },
+    // create DTrader account
+    // TODO: uncomment this when DTrade account will be added to wallet
```

Comment:
Note: Current wallet doesn't check wallet trading account (DTrade account). Will be uncommented later and set correct target class